### PR TITLE
added time-series example to readme

### LINKED ISSUE #647: Add explain time series example in README.md
The [Getting started section](https://github.com/dianna-ai/dianna#getting-started) of the README  a time-series example is missing!
We should add one time-series example.

### PR DESCRIPTION
**Pull request by cpranav93**
Adds a time-series example to the readme.

Visualization is not shown in example due to its complexity. The users are referred to the tutorial for an example there.

Closes #647

### CODE REVIEW

**Review by cwmeijer — CHANGES_REQUESTED** [review]
Looks great! Good call to add a link to a tutorial!

File: README.md [hunk 1]
```diff
@@ -118,42 +121,84 @@ In the library's documentation, the general usage is explained in [How to use DI
...
+num_features = len(timeseries_instance ) # The number of features to include in the explanation.
+num_samples = 500 # The number of samples to generate for the LIME explainer.
+num_slices = len(timeseries_instance) # The number of slices to divide the time series data into.
+```
+
+Which of your model's classes do you want an explanation for?
+
+```python
+class_names= [class_a, class_b] # The list of class names.
+labels = np.argsort(class_names) # The list of labels for different classes.
```
[the remark this answers is not in the document]
Comment (cpranav93) [reply README.md:188]: I believe the difference between the class_names and labels is not about the complete list and the interesting list but rather the string names in class_names and corresponding integers in labels. Would this make it clearer:
```suggestion
Which of your model's classes do you want an explanation for?

```python
class_names= [class_a, class_b] # String representation of the different classes of interest
labels = np.argsort(class_names) # Numerical representation of the different classes of interest for the model
```

File: README.md [hunk 2]
```diff
@@ -118,42 +121,84 @@ In the library's documentation, the general usage is explained in [How to use DI
...
 
+### Time-series example:
+
+```python
+model_path = 'your_model.onnx'  # model trained on images
+timeseries_instance = pd.read_csv('your_data_instance.csv').astype(float)
+
+num_features = len(timeseries_instance ) # The number of features to include in the explanation.
+num_samples = 500 # The number of samples to generate for the LIME explainer.
+num_slices = len(timeseries_instance) # The number of slices to divide the time series data into.
```
[the remark this answers is not in the document]
Comment (cpranav93) [reply README.md:181]: I am discussing with Yang about the necessity for num_slices. It is currently there in the input argument but is not used anywhere in the function.

**Review by cwmeijer — APPROVED** [review]
Much better! Thanks!

**Pull request merged**